api/fma/v1alpha1/launcherpoolpolicy_types.go

```diff
@@ -0,0 +1,222 @@
+/*
+Copyright 2025 The llm-d Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// LauncherPoolPolicy defines the proactive provisioning policy for the idle launcher pods.
+// The LauncherPoolPolicy semantics should be defined as below:
+//
+// ## Multiple LauncherPoolPolicy Objects
+//
+// 1. **Additive Semantics**
+//   - Multiple LauncherPoolPolicy objects follow additive semantics
+//   - All policies across all objects are evaluated together
+//   - Rules from different objects are combined to form the complete policy set
+//
+// 2. **Zero Objects Behavior**
+//   - When no LauncherPoolPolicy objects exist, no proactive provisioning occurs
+//   - System falls back to on-demand launcher pod creation
+//
+// ## Multiple LauncherPoolForNodeType Matching
+//
+// 1. **Multiple Matches**
+//   - When multiple LauncherPoolForNodeType structs match a single Node (across same or different LauncherPoolPolicy objects):
+//     - For each unique combination of (Node, Accelerator, LauncherConfig), select the highest LauncherCount value
```

Comment:
I think "Accelerator set" is meant here rather than "Accelerator".

Reply:
make sense